From 3236bde68ae85008d4b802a7e0624c9d0671dd09 Mon Sep 17 00:00:00 2001
From: Ran Shalit <rshalit@habana.ai>
Date: Fri, 09 Apr 2021 13:02:56 +0300
Subject: [PATCH] [SW-36011] gaudi: fix rr index for spsram

Use RR_23 for protetion of SP SRAM, instead off
using RR_17, which is already used in dynamic RR.

Change-Id: I68144deb3a653d74d52ae27d891878dc260457cd
Signed-off-by: Ran Shalit <rshalit@habana.ai>
---

diff --git a/gaudi/security.c b/gaudi/security.c
index bdc1083..ff38eb6 100644
--- a/gaudi/security.c
+++ b/gaudi/security.c
@@ -1515,12 +1515,12 @@
 	 * PCIe IF HBW RRs are for direct access from PCIe IF to PSOC domain:
 	 * STM, SPI Flash, Scratchpad SRAM, PCIe FW SRAM.
 	 * SPSRAM space for fit is used in dynamic RR.
-	 * Assign the PCIe IF RR_16 , RR_17 (0-15 and 28-31 are allocated for
+	 * Assign the PCIe IF RR_16 , RR_23 (0-15 and 28-31 are allocated for
 	 * LBW) for this job.
 	 * NOTE: The PCIe RRs use min/max concept and HIT is: min < hit < max
 	 */
 	preboot_secure_pcie_rr(16, mmSTM_BASE - 1, FIT_IMG_SPSRAM_BASE);
-	preboot_secure_pcie_rr(17, FIT_IMG_SPSRAM_END - 1, mmRAZWI_BASE);
+	preboot_secure_pcie_rr(23, FIT_IMG_SPSRAM_END - 1, mmRAZWI_BASE);
 }
 
 static void preboot_security_pci_rtr_disable(int rr)
